test/integration/aaa_download_only_test.go

```diff
@@ -199,18 +197,19 @@ func TestDownloadOnlyDocker(t *testing.T) {
 	}
 
 	// Make sure this image exists in the docker daemon
-	ref, err := name.ParseReference(kic.BaseImage)
+	images, err := exec.Command("docker", "images", "--format", "{{.Repository}}:{{.Tag}}@{{.Digest}}").Output()
 	if err != nil {
-		t.Errorf("parsing reference failed: %v", err)
+		t.Errorf("getting list of docker images failed: %v\nOutput: %s", err, string(images))
 	}
-	if _, err := daemon.Image(ref); err != nil {
-		t.Errorf("expected image does not exist in local daemon: %v", err)
+	want := kic.BaseImage
+	if !strings.Contains(string(images), want) {
+		t.Errorf("expected image does not exist in local daemon; got:\n%s wanted:\n%s", string(images), want)
 	}
 }
 
 func runningDockerDriver(startArgs []string) bool {
```

Comment:
please move this to main.go 

and name it DockerDriver()
same as https://github.com/kubernetes/minikube/blob/82900a7811e479755e03ae020ebb44152dfd93d8/test/integration/main.go#L63
```
// NoneDriver returns whether or not this test is using the none driver
func NoneDriver() bool {
	return strings.Contains(*startArgs, "--driver=none")
}
```